spec/swagger_helper.rb

```diff
@@ -205,6 +205,18 @@
               },
             },
           },
+          create_vacancy_response: {
+            type: :object,
+            properties: {
+              id: { type: :string, example: "9d8f5715-2e7c-4e64-8e34-35f510c12e66" },
+              errors: {
+                type: :array,
+                items: { type: :string },
+                nullable: true,
+              },
+            },
+            required: %w[id],
```

Comment:
if errors are possible, then id can't be required. Maybe this should be oneOf id an errors?

Reply:
oneOf - an object with an 'id' property or an object with an 'errors' property